### DISCUSSION TITLE
chia plots check  report "ERROR"

### DISCUSSION THREAD

**Discussion by uldh** [madMAx43v3r/chia-plotter#644] [Q&A]
Does anyone know why "chia plots check" report those "ERROR" ? i have 1000 plots ERROR reported, but those 1000 plots works well now.
https://user-images.githubusercontent.com/60197354/120218073-371ae080-c207-11eb-9ec3-3690dc2fbe1a.png

**Comment by madMAx43v3r** [comment]
usually happens during copy of plot
